DataFormats/MuonSeed/src/classes_def.xml

```diff
@@ -1,5 +1,6 @@
 <lcgdict>
-      <class name="L2MuonTrajectorySeed"  ClassVersion="12">
+      <class name="L2MuonTrajectorySeed"  ClassVersion="13">
+       <version ClassVersion="13" checksum="2121764882"/>
```

Comment:
I just noticed that the class L2MuonTrajectorySeed is updated to ClassVersion="13", but L3MuonTrajectorySeed is not. Is that intentional?

Reply:
Yes, the `L2MuonTrajectorySeed` class was updated to contain a reference to the L1Tk Muon used to produce the seed (and the corresponding constructor), while this is not the case for  `L3MuonTrajectorySeed`